connectors/kafka/src/main/java/forklift/connectors/KafkaConnector.java

```diff
@@ -0,0 +1,133 @@
+package forklift.connectors;
+
+import forklift.consumer.ForkliftConsumerI;
+import forklift.consumer.KafkaTopicConsumer;
+import forklift.controller.KafkaController;
+import forklift.message.MessageStream;
+import forklift.producers.ForkliftProducerI;
+import forklift.producers.KafkaForkliftProducer;
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.ProducerConfig;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import java.util.Properties;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Manages both consuming and producing events on the kafka message broker.
+ */
+public class KafkaConnector implements ForkliftConnectorI {
+    private static final Logger log = LoggerFactory.getLogger(KafkaConnector.class);
+    private final String kafkaHosts;
```

Comment:
Minor nit: These fields could be separated a little better (i.e separate values from state)